aep/general/0004/aep.md

```diff
@@ -16,65 +16,99 @@ tools such as Kubernetes or GraphQL interact with APIs from multiple providers.
 APIs **must** define a resource type for each resource in the API, according to
 the following pattern: `{API Name}/{Type Name}`. The type name:
 
-- **must** Start with an uppercase letter.
-- **must** Only contain alphanumeric characters.
+- **must** Only contain ASCII alphanumeric characters.
+- **must** Start with a lowercase letter.
 - **must** Be of the singular form of the noun.
-- **must** Use PascalCase (UpperCamelCase).
-- For Kubernetes, the type name **must** match the [object][] name.
-- For OpenAPI, the type name **must** match the name of the schema representing
-  the object.
-- For Protobuf, the type name **must** match the name of the protobuf message.
+- **must** Use kebab case.
+- For Kubernetes, the type name when converted to upper camel case **must**
+  match the [object][] name.
+- For OpenAPI, the type name when converted to upper camel case **must** match
+  the name of the schema representing the object.
+- For protobuf, the type name when converted to upper camel case **must** match
+  the name of the protobuf message.
 
 ### Examples
 
 Examples of resource types include:
 
-- `pubsub.example.com/Topic`
-- `pubsub.example.com/Subscription`
-- `spanner.example.com/Database`
-- `spanner.example.com/Instance`
-- `networking.istio.io/Instance`
+- `networking.istio.io/instance`
+- `pubsub.example.com/topic`
+- `pubsub.example.com/subscription`
+- `spanner.example.com/database`
+- `spanner.example.com/instance`
+- `user.example.com/user-event`
```

Comment:
What about the `apis.example.com/users/user-event` form?  I don't want to argue for that to be _the_ canonical form, but I do think it should be supported.  Otherwise we are prescribing that everyone has to use a subdomain for each API, which raises the barrier to entry (and would have been a dealbreaker for Roblox, for example).

Reply:
We do have an outstanding issue to pick up this discussion at some point: https://github.com/aep-dev/aep.dev/issues/74#issue-2000272320.

I'll update the example as suggested, but my main concern is we have to go put these types of examples everywhere - ideally we'd consolidate discussion of a single concept to a single AEP so we don't have to go rewrite a ton of examples as things change.